src/core/consent/createConsentStateMachine.js

```diff
@@ -111,6 +111,22 @@ export default ({ logger }) => {
     awaitConsent: awaitInitial,
     withConsent() {
       return this.awaitConsent(true);
+    },
+    current() {
+      switch (this.awaitConsent) {
+        case awaitInDefault:
+          return { state: "in", wasSet: false };
```

Comment:
Is the goal to differentiate between default consent and set consent by the visitor?

Reply:
Yes. We only want to show containers when we have a "out" cookie.